src/platform/testing/e2e/nightwatch-commands/repeatKeypress.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable func-names, prefer-arrow-callback */
+
+/**
+ * Allow test writers to build a shorthand for multiple keypresses.
+ * This is a common part of keyboard navigation, especially when
+ * users have to press TAB several times to move through a group
+ * of checkboxes or form elements.
+ *
+ * ```javascript
+ *  this.demoTest = function (client) {
+ *    client.repeatKeypress(client.Keys.TAB, 3);
+ *  };
+ * ```
+ *
+ * @method repeatKeypress
+ * @param {object} key The Selenium key to press. Formatted Client.Keys.KEY.
+ * @param {number} pressCount Number of times to press the key.
+ * @api commands
+ */
+module.exports.command = function repeatKeypress(key, pressCount) {
```

Comment:
I'd recommend adding another optional argument for `waitTime` and give it a default of 125 rather than just hardcoding that bit in there.